modules/effects/arrow-effects-kotlinx-coroutines/src/main/kotlin/arrow/effects/DeferredK.kt

```diff
@@ -496,40 +496,48 @@ sealed class DeferredK<A>(
       CompletableDeferred<A>().apply { completeExceptionally(t) }.k()
 
     /**
-     * Starts a coroutine that'll run [Proc].
+     * Starts a coroutine that'll run [DeferredKProc].
      *
-     * Matching the behavior of [async],
+     * Matching the behavior of [asyncK],
      * its [CoroutineContext] is set to [DefaultDispatcher]
      * and its [CoroutineStart] is [CoroutineStart.LAZY].
      *
      * {: data-executable='true'}
      *
      * ```kotlin:ank
```

Comment:
Do we need Ank for this when it has no return?

Reply:
cc @raulraja